### PR TITLE
Translate Portugues pt-BR

### PR DESCRIPTION
**Pull request by herickf**
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)

## Proposed changes

Create the Portuguese pt-BR translation following the standard and en files.

## Checklist

- [X] I have read the [contribution guidelines](https://github.com/node-red/node-red/blob/master/CONTRIBUTING.md)
- [ ] For non-bugfix PRs, I have discussed this change on the forum/slack team.
- [ ] I have run `grunt` to verify the unit tests pass
- [ ] I have added suitable unit tests to cover the new/changed functionality

### CODE REVIEW

**Comment by hardillb** [comment]
@herickf I've just checked your commit and it was not made with the herick2008@gmail.com email address, but with the xxxxx@eletasolucoes.com.br address, you need to sign the CLA with the same email address that is attached to the commit.

File: packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json [hunk 1]
```diff
@@ -0,0 +1,1125 @@
...
+            "conflictNotification1": "Alguns dos nós que você está importando já existem em sua área de trabalho.",
+            "conflictNotification2": "Selecione quais nós importar e se deseja substituir os nós existentes ou importar uma cópia deles."
+        },
+        "copyMessagePath": "Caminho copiado",
+        "copyMessageValue": "Valor copiado",
+        "copyMessageValue_truncated": "Valor truncado copiado"
+    },
+    "deploy": {
+        "deploy": "Implantar",
+        "full": "Cheio",
```
Comment (machadotiago) [inline packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json:262]: Change to "Completo"

File: packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json [hunk 2]
```diff
@@ -0,0 +1,1125 @@
...
+            "conflict": "O servidor está executando um conjunto de fluxos mais recente.",
+            "backgroundUpdate": "Os fluxos no servidor foram atualizados.",
+            "conflictChecking": "Verificando se as alterações podem ser mescladas automaticamente",
+            "conflictAutoMerge": "As alterações não incluem conflitos e podem ser mescladas automaticamente.",
+            "conflictManualMerge": "As mudanças incluem conflitos que devem ser resolvidos antes de serem implantados.",
+            "plusNMore": "+ __count__ mais"
+        }
+    },
+    "eventLog": {
+        "title": "Event log",
```
Comment (machadotiago) [inline packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json:301]: Change to "Registro de eventos"

File: packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json [hunk 3]
```diff
@@ -0,0 +1,1125 @@
...
+        "view": "View log"
```
Comment (machadotiago) [inline packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json:302]: Change to "Visualizar eventos"

File: packages/node_modules/@node-red/nodes/locales/pt-BR/messages.json [hunk 4]
```diff
@@ -0,0 +1,1048 @@
...
+        "toStatus": "status do nó (32 caracteres)",
+        "severity": "Nível",
+        "notification": {
+            "activated": "Ativado com sucesso: __label__",
+            "deactivated": "Desativado com sucesso: __label__"
+        },
+        "sidebar": {
+            "label": "depurar",
+
+            "name": "mensagens de depuração",
```
Comment (machadotiago) [inline packages/node_modules/@node-red/nodes/locales/pt-BR/messages.json:143]: Change to "Mensagens de depuração"

File: packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json [hunk 5]
```diff
@@ -0,0 +1,1125 @@
...
+        "groupCopied": "__count__ grupo copiado",
+        "groupCopied_plural": "__count__ grupos copiados",
+        "groupStyleCopied": "Estilo de grupo copiado",
+        "invalidFlow": "Fluxo inválido: __message__",
+        "recoveredNodes": "Nós recuperados",
+        "recoveredNodesInfo": "Os nós neste fluxo não tinham um ID de fluxo válido quando foram importados. Eles foram adicionados a este fluxo para que você possa restaurá-los ou excluí-los.",
+        "recoveredNodesNotification": "<p>Nós importados sem um ID de fluxo válido </p> <p> Eles foram adicionados a um novo fluxo chamado '__flowName__'.</p>",
+        "export": {
+            "selected": "nós selecionados",
+            "current": "fluxo corrente",
```
Comment (machadotiago) [inline packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json:230]: Change to "fluxo atual"

File: packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json [hunk 6]
```diff
@@ -0,0 +1,1125 @@
...
+        "errors": {
+            "cannotCreateDiffGroups": "Não é possível criar grupo usando nós de grupos diferentes",
+            "cannotAddSubflowPorts": "Não é possível adicionar portas de subfluxo a um grupo"
+        }
+    },
+    "editor": {
+        "configEdit": "Editar",
+        "configAdd": "Adicioanr",
+        "configUpdate": "Atualizar",
+        "configDelete": "Deletar",
```
Comment (machadotiago) [inline packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json:379]: Change to "Remover"

File: packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json [hunk 7]
```diff
@@ -0,0 +1,1125 @@
...
+                "resetTheEncryptionKey": "Reset the encryption key",
+                "changeTheEncryptionKey": "Redefina a chave de criptografia",
+                "currentKey": "Chave atual",
+                "newKey": "Nova chave",
+                "credentialsAlert": "Isso excluirá todas as credenciais existentes",
+                "versionControl": "Controle de versão",
+                "branches": "Branches",
+                "noBranches": "Sem branches",
+                "deleteConfirm": "Tem certeza de que deseja excluir o branch local '__name__'? Isto não pode ser desfeito.",
+                "unmergedConfirm": "O branch local '__name__' tem alterações não mescladas que serão perdidas. Tem certeza que deseja deletar?",
```
Comment (machadotiago) [inline packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json:730]: Change to "O branch local '__name__' tem alterações não mescladas que serão perdidas. Tem certeza que deseja remover estas alterações?"

File: packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json [hunk 8]
```diff
@@ -0,0 +1,1125 @@
...
+                "addRemote": "adicionar remote",
+                "addRemote2": "Adicionar remote",
+                "remoteName": "Nome do remote",
+                "nameRule": "Deve conter apenas A-Z 0-9 _ -",
+                "url": "URL",
+                "urlRule": "https://, ssh:// ou file://",
+                "urlRule2": "Não inclua o nome de usuário / senha no URL",
+                "noRemotes": "Sem remote",
+                "deleteRemoteConfrim": "Tem certeza de que deseja excluir remote '__name__'?",
+                "deleteRemote": "Deletar remote"
```
Comment (machadotiago) [inline packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json:742]: Change to "Remover remoto"

File: packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json [hunk 9]
```diff
@@ -0,0 +1,1125 @@
...
+                "nameRule": "Deve conter apenas A-Z 0-9 _ -",
+                "passphrase": "Passphrase",
+                "passphraseShort": "Passphrase muito curta",
+
+                "optional": "Opcional",
+                "cancel": "Cancelar",
+                "generate": "Gerar chave",
+                "noSshKeys": "Sem chaves SSH",
+                "copyPublicKey": "Copiar chave pública para a área de transferência",
+                "delete": "Deletar chave key",
```
Comment (machadotiago) [inline packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json:770]: Change to "Remover chave"

File: packages/node_modules/@node-red/nodes/locales/pt-BR/messages.json [hunk 10]
```diff
@@ -0,0 +1,1048 @@
...
+            "utf8String": "string UTF8",
+            "utf8String_plural": "strings UTF8",
+            "binaryBuffer": "buffer binário",
+            "binaryBuffer_plural": "buffers binários",
+            "allProps": "inclui todas as propriedades existentes em cada mensagem"
+        },
+        "action": {
+            "append": "anexar ao arquivo",
+            "overwrite": "sobrescrever file",
+            "delete": "deletar file"
```
Comment (machadotiago) [inline packages/node_modules/@node-red/nodes/locales/pt-BR/messages.json:880]: Change to "Remover arquivo"

File: packages/node_modules/@node-red/nodes/locales/pt-BR/messages.json [hunk 11]
```diff
@@ -0,0 +1,1048 @@
...
+            "iso": "páginas de código ISO",
+            "ibm": "páginas de código IBM",
+            "mac": "páginas de código Mac",
+            "koi8": "páginas de código KOI8",
+            "misc": "Diversos"
+        },
+        "errors": {
+            "nofilename": "Nenhum nome de arquivo especificado",
+            "invaliddelete": "Aviso: exclusão inválida. Use a opção de exclusão específica na caixa de diálogo de configuração.",
+            "deletefail": "falhou ao deletar o arquivo: __error__",
```
Comment (machadotiago) [inline packages/node_modules/@node-red/nodes/locales/pt-BR/messages.json:912]: Change to "falhou ao remover o arquivo: __error__"

File: packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json [hunk 12]
```diff
@@ -0,0 +1,1125 @@
...
+        "enabled": "Habilitar",
+        "disabled": "Desabilitar",
+        "info": "Descrição",
+        "selectNodes": "Clique nos nós para selecionar"
+    },
+    "menu": {
+        "label": {
+            "view": {
+                "view": "Visão",
+                "grid": "Rede",
```
Comment (machadotiago) [inline packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json:68]: Change to "Grade"

File: packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json [hunk 13]
```diff
@@ -0,0 +1,1125 @@
...
+        "functions": "Funções",
+        "functionReference": "Referência de função",
+        "insert": "Inserir",
+        "title": "JSONata Expression editor",
+        "test": "Teste",
+        "data": "Mensagem de exemplo",
+        "result": "Resultado",
+        "format": "formato da expressão",
+        "compatMode": "Modo de compatibilidade habilitado",
+        "compatModeDesc": "<h3> Modo de compatibilidade JSONata </h3> <p> A expressão atual parece ainda fazer referência a <code> msg </code>, então será avaliada no modo de compatibilidade. Atualize a expressão para não usar <code > msg </code>, pois este modo será removido no futuro. </p> <p> Quando o suporte JSONata foi adicionado pela primeira vez ao Node-RED, era necessária a expressão para fazer referência ao objeto <code> msg </code> . Por exemplo, <code> msg.payload </code> seria usado para acessar a carga. </p> <p> Isso não é mais necessário, pois a expressão será avaliada em relação à mensagem diretamente. Para acessar a carga, o expressão deve ser apenas <code> carga útil </code>. </p> ",
```
Comment (machadotiago) [inline packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json:883]: Change to:
`<h3> Modo de compatibilidade JSONata </h3> <p> A expressão atual parece ainda fazer referência a <code> msg </code>, então será avaliada no modo de compatibilidade. Atualize a expressão para não usar <code > msg </code>, pois este modo será removido no futuro. </p> <p> Quando o suporte JSONata foi adicionado pela primeira vez ao Node-RED, era necessária a expressão para fazer referência ao objeto <code> msg </code> . Por exemplo, <code> msg.payload </code> seria usado para acessar o conteúdo. </p> <p> Isso não é mais necessário, pois a expressão será avaliada em relação à mensagem diretamente. Para acessar a o conteúdo, a expressão deve ser apenas <code>payload</code>. </p>`

File: packages/node_modules/@node-red/nodes/locales/pt-BR/common/20-inject.html [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<script type="text/html" data-help-name="inject">
+<p>Injeta uma mensagem em um fluxo manualmente ou em intervalos regulares. A mensagem
```
Comment (machadotiago) [inline packages/node_modules/@node-red/nodes/locales/pt-BR/common/20-inject.html:18]: Change to 
`<p>Injeta uma mensagem em um fluxo manualmente ou em intervalos regulares. O conteúdo da`

File: packages/node_modules/@node-red/nodes/locales/pt-BR/common/20-inject.html [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+  a carga útil pode ser de vários tipos, incluindo strings, objetos JavaScript ou a hora atual.</p>
```
Comment (machadotiago) [inline packages/node_modules/@node-red/nodes/locales/pt-BR/common/20-inject.html:19]: Change to:
`mensagem pode ser de vários tipos, incluindo strings, objetos JavaScript ou a hora atual.</p>`

File: packages/node_modules/@node-red/nodes/locales/pt-BR/common/20-inject.html [hunk 16]
```diff
@@ -0,0 +1,40 @@
...
+<h3>Saídas</h3>
+<dl class="message-properties">
+    <dt>payload<span class="property-type">vários</span></dt>
+    <dd>A carga configurada da mensagem.</dd>
```
Comment (machadotiago) [inline packages/node_modules/@node-red/nodes/locales/pt-BR/common/20-inject.html:23]: Change to:
`    <dd>O conteúdo configurado da mensagem.</dd>`

File: packages/node_modules/@node-red/nodes/locales/pt-BR/common/20-inject.html [hunk 17]
```diff
@@ -0,0 +1,40 @@
...
+    <dt class="optional">topic <span class="property-type">string</span></dt>
+    <dd>Uma propriedade opcional que pode ser configurada no nó.</dd>
+</dl>
+<h3>Detalhes</h3>
+<p>O nó Inject pode iniciar um fluxo com um valor de carga útil específico.
```
Comment (machadotiago) [inline packages/node_modules/@node-red/nodes/locales/pt-BR/common/20-inject.html:28]: Change to:
`<p>O nó Inject pode iniciar um fluxo com um conteúdo específico.`

File: packages/node_modules/@node-red/nodes/locales/pt-BR/common/20-inject.html [hunk 18]
```diff
@@ -0,0 +1,40 @@
...
+A carga útil padrão é um carimbo de data / hora da hora atual em milissegundos desde 1º de janeiro de 1970. </p>
```
Comment (machadotiago) [inline packages/node_modules/@node-red/nodes/locales/pt-BR/common/20-inject.html:29]: Change to:
`O conteúdo padrão é um estampa de data / hora da momento atual em milisegundos desde 1º de janeiro de 1970. </p>`

File: packages/node_modules/@node-red/nodes/locales/pt-BR/common/20-inject.html [hunk 19]
```diff
@@ -0,0 +1,40 @@
...
+<p> O nó também oferece suporte à injeção de strings, números, booleanos, objetos JavaScript ou valores de fluxo / contexto global. </p>
+<p> Por padrão, o nó é disparado manualmente clicando em seu botão no editor. Também pode ser definido como
+injete em intervalos regulares ou de acordo com um cronograma. </p>
+<p> Ele também pode ser configurado para injetar uma vez cada vez que os fluxos forem iniciados. </p>
+<p> O <i> Intervalo </i> máximo que pode ser especificado é de cerca de 596 horas / 24 dias. No entanto, se você estiver olhando em intervalos
+mais de um dia, você deve considerar o uso de um nó de agendador que pode lidar com quedas de energia e reinicializações. </p>
+<p> <b> Observação </b>: As opções de <i> "Intervalo entre os tempos" </i> e <i> "em um horário específico" </i> usam o sistema cron padrão.
+Isso significa que 20 minutos serão na próxima hora, 20 minutos atrás e 40 minutos depois - não em 20 minutos.
+Se você quiser a cada 20 minutos a partir de agora - use a opção <i> "intervalo" </i>. </p>
+<p> <b> Observação </b>: para incluir uma nova linha em uma string, você deve usar um nó de função para criar a carga útil. </p>
```
Comment (machadotiago) [inline packages/node_modules/@node-red/nodes/locales/pt-BR/common/20-inject.html:39]: Change to:
`<p> <b> Observação </b>: para incluir uma nova linha em uma string, você deve usar um nó de função para alterar o conteúdo. </p>`

File: packages/node_modules/@node-red/nodes/locales/pt-BR/common/25-status.html [hunk 20]
```diff
@@ -0,0 +1,34 @@
...
+      <dd> o texto do status. </dd>
+      <dt> status.source.type <span class = "property-type"> string </span> </dt>
+      <dd> o tipo de nó que relatou o status. </dd>
+      <dt> status.source.id <span class = "property-type"> string </span> </dt>
+      <dd> o id do nó que relatou o status. </dd>
+      <dt> status.source.name <span class = "property-type"> string </span> </dt>
+      <dd> o nome, se definido, do nó que relatou o status. </dd>
+  </dl>
+  <h3> Detalhes </h3>
+ <p> Este nó não produz uma <code> carga útil </code>. </p>
```
Comment (machadotiago) [inline packages/node_modules/@node-red/nodes/locales/pt-BR/common/25-status.html:31]: Change to:
` <p> Este nó não produz um <code>payload</code>. </p>`

File: packages/node_modules/@node-red/nodes/locales/pt-BR/function/16-range.html [hunk 21]
```diff
@@ -0,0 +1,44 @@
...
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<script type="text/html" data-help-name="range">
+<p> Mapeia um valor numérico para um intervalo diferente. </p>
+    <h3> Entradas </h3>
+    <dl class = "message-properties">
+        <dt> carga útil <span class = "property-type"> número </span> </dt>
```
Comment (machadotiago) [inline packages/node_modules/@node-red/nodes/locales/pt-BR/function/16-range.html:21]: <dt>payload<span class = "property-type"> número </span> </dt>

File: packages/node_modules/@node-red/nodes/locales/pt-BR/function/16-range.html [hunk 22]
```diff
@@ -0,0 +1,44 @@
...
+        <dd> A carga útil <i> deve </i> ser um número. Qualquer outra coisa tentará ser
```
Comment (machadotiago) [inline packages/node_modules/@node-red/nodes/locales/pt-BR/function/16-range.html:22]: Change to:
`        <dd> O payload <i> deve </i> ser um número. Qualquer outra coisa tentará ser`

File: packages/node_modules/@node-red/nodes/locales/pt-BR/function/16-range.html [hunk 23]
```diff
@@ -0,0 +1,44 @@
...
+        analisado em um número e rejeitado se falhar. </dd>
+    </dl>
+    <h3> Saídas </h3>
+    <dl class = "message-properties">
+        <dt> carga útil <span class = "property-type"> número </span> </dt>
```
Comment (machadotiago) [inline packages/node_modules/@node-red/nodes/locales/pt-BR/function/16-range.html:27]: Change to:
`        <dt> payload <span class = "property-type"> número </span> </dt>`

File: packages/node_modules/@node-red/nodes/locales/pt-BR/function/80-template.html [hunk 24]
```diff
@@ -0,0 +1,51 @@
...
+  </dl>
+  <h3> Detalhes </h3>
+  <p> Por padrão, usa o <i> <a href="http://mustache.github.io/mustache.5.html" target="_blank"> bigode </a> </i>
+  formato, mas pode ser desativado, se necessário. </p>
+  <p> Por exemplo, quando um modelo de:
+  <pre> Olá {{payload.name}}. Hoje é {{date}} </pre>
+  <p> recebe uma mensagem contendo:
+  <pre> {
+data: "segunda-feira",
+carga útil: {
```
Comment (machadotiago) [inline packages/node_modules/@node-red/nodes/locales/pt-BR/function/80-template.html:40]: Change to:
`payload: {`

**Review by machadotiago — CHANGES_REQUESTED** [review]
I fixed some translations and apart from that I have some comments: 
- The correct translation for payload, in this context, is "conteúdo", not "carga útil", but you cannot translate this term in some cases, specially when it is an example like "<code>payload</code>". I fixed it in **some** nodes like "Inject", "Change" and "Range" but there are more 60 other occurrences to be fixed, can you have a look on that?
- The word "Deletar" should be avoided, the correct term is "Remover", I think that I cover all occurrences in my comments.
- Great job :)

**Comment by FabsMuller** [comment]
Herick anf Tiago, my .50 cents here. I just saw your pull request today. The work is quite good but in my opinion not for a pull request yet. Besides Tiago observations, there is still some fine tuning to do. So before submitting the PR, we could discuss a lit bit some translations in order to pr good work. If you don't mind I think we can improve and call others for help. For example in my opinion Carga útil is a correct translation for Payload because it really what the word means in PT and in Node-red.  I also saw "passphrase", "branch" and "remote" that could be translated to: frase de passe, Ramo e remote, but I didn't saw the whole work. Again it's a great work and a great start but let's discuss it before pr so we can submit a wonderful text. What do you think?

**Comment by FabsMuller** [comment]
Hello Guys , just opened a new PR #3804 based on V.3.0.1 . I asked at Node-Red and Nick answered to submit a new PR to be more fresh to the lastest version. So this PR can be closed and lest work on the new one. Just to clear some thoughts, I'm very fan of Iberic (Portugal/Spain) approach of translating everything. So in the New PR you will find almost everything translated. Even if you don't agree or are not used to some of them, pls let's discuss it later. But I liked the results very much. Brgds.

**Comment by Steve-Mcl** [comment]
Closing in favour of https://github.com/node-red/node-red/pull/3804

**Pull request closed without merging**